src/VecSim/vec_sim_common.h

```diff
@@ -111,6 +111,7 @@ typedef struct {
     };
     size_t batchSize;
     VecSearchMode searchMode;
+    void *timeoutCtx;
```

Comment:
To document that this isn't passed by the user and therefore doesn't need to be resolved as the others